src/styles/index.scss

```diff
@@ -1,3 +1,124 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  &__img {
+    padding: 0 3px;
+    &-img {
+      display: block;
+      background-image: url('../images/imac.jpeg');
+      background-size: cover;
+      height: 134px;
+    }
+  }
+
+  &__header {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: #060b35;
+    margin: 0;
+    padding-top: 40px;
+    text-transform: bold;
```

Comment:
The property 'text-transform: bold' is not valid. You might want to use 'font-weight: bold' instead to achieve bold text styling.